packages/asset-swapper/src/utils/rfq_maker_blacklist.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Tracks a maker's history of timely responses, and manages whether a given
+ * maker should be avoided for being too latent.
+ */
+export class RfqMakerBlacklist {
+    // tslint:disable-next-line:custom-no-magic-numbers
+    private static readonly _msPerMinute = 1000 * 60;
+    private readonly _makerTimeoutStreakLength: { [makerUrl: string]: number } = {};
+    private readonly _makerBlacklistedUntilDate: { [makerUrl: string]: number } = {};
+    constructor(private readonly _blacklistDurationMinutes: number, private readonly _timeoutStreakThreshold: number) {}
```

Comment:
why do constructor args have underscores? I know you are taking a shortcut here which I took in the past, but I was suggested to simply set up constructor the old fashioned way

Reply:
the underscores are there because the linter required me to put them there, because they're not just function arguments, they're also declarations of member variables.

Reply:
right, could we use the old fashioned way instead? or do u think it's not worth it? underscore arguments usually are an alias for unused arguments